src/tests/ftest/cart/test_group_np_cli.c

```diff
@@ -81,7 +81,10 @@ test_run(void)
 
 			for (tag = 0; tag < test_g.t_srv_ctx_num; tag++) {
 				DBG_PRINT("Sending rpc to %d:%d\n", rank, tag);
-				check_in(grp, rank, tag);
+
+        for (j = 0; j < test_g.t_num_checkins_to_send; j++) {
```

Comment:
(style)  code indent should use tabs where possible
(style)  suspect code indent for conditional statements (8, 10)